app/services/doc_auth/processed_alert_to_log_alert_formatter.rb

```diff
@@ -0,0 +1,34 @@
+module DocAuth
+  class ProcessedAlertToLogAlertFormatter
```

Comment:
I think we may want to throw a comment up here describing what this is doing. I'm normally not a fan of comments but this one is a little sophisticated